aviary/docs/examples/coupled_aircraft_mission_optimization.ipynb

```diff
@@ -145,8 +160,7 @@
    "execution_count": null,
    "metadata": {
     "tags": [
-     "remove-input",
-     "remove-output"
+     "remove_cell"
```

Comment:
Something odd going on with these tags here. We have a "remove_cell" and a "remove-cell" tag. Are they both valid?

Reply:
should be remove-cell, I'll fix this